lib/analyzer.js

```diff
@@ -214,11 +222,10 @@ Analyzer.prototype._parseHTML = function _parseHTML(htmlImport,
   var depHrefs = [];
   var metadataLoaded = Promise.resolve(EMPTY_METADATA);
   var parsed;
-
   try {
-    parsed = importParse(htmlImport);
+    parsed = importParse(htmlImport, href);
   } catch (err) {
-    console.log(err);
+    console.log(err.stack);
```

Comment:
An error that's not `catch()`ed should definitely bubble up as an uncaught exception, otherwise Promises are just broken. I'll look for docs to verify.